docs/semgrep-sc/sc-notifications.md

```diff
@@ -0,0 +1,93 @@
+---
+slug: receiving-notifications-from-ssc
+append_help_link: true
+description: "Receive notifications about reachable findings from your Semgrep Supply Chain scans."
+tags:
+    - Semgrep Supply Chain
+    - Team & Enterprise Tier
+title: Receiving notifications from SSC scans
+hide_title: true
+---
+
+import MoreHelp from "/src/components/MoreHelp"
+
+<ul id="tag__badge-list">
+{
+Object.entries(frontMatter).filter(
+    frontmatter => frontmatter[0] === 'tags')[0].pop().map(
+    (value) => <li class='tag__badge-item'>{value}</li> )
+}
+</ul>
+
+# Receiving notifications from Semgrep Supply Chain (SSC) scans
+
+Developers can be notified of vulnerabilities in their GitHub or GitLab environment through pull request (PR) or merge request (MR) comments.
+
+## Receiving SSC notifications through PR or merge request MR comments
+
+![Semgrep Supply Chain PR comment](/img/ssc-pr-comment.png#bordered)
+_Figure 1_. Screenshot of a PR comment from SSC in a GitHub repository.
+
+Semgrep Supply Chain can send pull request (PR) or merge request (MR) comments to notify developers in their GitHub or GitLab environment of **third-party reachable vulnerabilities** detected by a Semgrep Supply Chain scan. The following information is provided:
+
+<dl>
+<dt>Risk</dt>
+<dd>A description of the vulnerability, including the types of attack it is vulnerable to.</dd>
+<dt>Fix</dt>
+<dd>Indicates what versions to upgrade to, if any, that resolves or eliminates the vulnerability.</dd>
+<dt>Reference</dt>
+<dd>A link to additional information about the vulnerability from <a href="https://github.com/advisories">GitHub Advisory Database</a> and the <a href="https://nvd.nist.gov/vuln">National Vulnerability Database (NVD)</a>, if available.</dd>
+</dl>
+
+:::info
+Pull or merge requests with vulnerabilities detected by SSC are **not** blocked from merging.
+:::
+
+
+### Enabling Semgrep Supply Chain to send PRs or MRs
+
+When you add or onboard a repository to Semgrep App, you typically provide permissions enabling Semgrep Supply Chain to send PR or MR comments to your repository without additional steps.
```

Comment:
Hmm the phrasing might be a bit off here -- how about like "Once you have onboard a repository to Semgrep App and start scanning, Semgrep Supply Chain will send PR or MR Comments --if any-- without any additional setup steps.

Reply:
Yeah I think I went to far in the "well maybe you won't receive comments without additional steps" (this is because some folks use a self-hosted github plan). I rewrote a heading for "custom semgrep setups" in case they don't work to help with troubleshooting. Here's a screenshot:

<img width="691" alt="ForAndy" src="https://user-images.githubusercontent.com/20766840/214267785-058b264a-68c1-4f44-996d-3f4fa026808a.png">